.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+node_modules
```

Comment:
Shouldn't need node_modules in this PR, because you haven't added any node modules.